resources/views/contact.blade.php

```diff
@@ -7,24 +7,43 @@
         <div class="row">
             <div class="col-md-12">
                 <h4>Форма обратной связи</h4>
-                <form action="./mail.php" method="post">
-                    <div class="form-group col-md-4">
-                        <label for="name">Ваше имя:</label>
-                        <input type="name" name="name" class="form-control" id="name" placeholder="Name">
+                @if ($errors->any())
+                    <div class="alert alert-danger">Error!</div>
+                @endif
+                <form action="{{ route('show') }}" method="post">
```

Comment:
некритично в данном случае, но наверное должно быть
```suggestion
                <form action="{{ route('contact_store') }}" method="post">
```